helm/charts/hpe-flexvolume-driver/templates/sc.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
- name: hpe-flexvolume-sc-test
+ name: hpe-flexvolume-sc
```

Comment:
It would be great if we can annotate the `StorageClass` to be the default. I'm not sure how this will work if there's already a default `StorageClass` on the cluster. I also think naming the class `hpe-generic` or `hpe-standard` would be more appropriate.

Reply:
ok, we can take defaultStorageClass=true/false option  from user and create a default one with annotation. we default to true, and user can override if they already have one..does that sound good? @rgcostea